proxy/src/main/java/com/wavefront/agent/listeners/ZipkinPortUnificationHandler.java

```diff
@@ -174,18 +175,28 @@ private void processZipkinSpan(zipkin2.Span zipkinSpan) {
     // Set spanName.
     String spanName = zipkinSpan.name() == null ? DEFAULT_SPAN_NAME : zipkinSpan.name();
 
+    String spanId = Utils.convertToUuidString(zipkinSpan.id());
+    String traceId = Utils.convertToUuidString(zipkinSpan.traceId());
     //Build wavefront span
     Span newSpan = Span.newBuilder().
         setCustomer("dummy").
         setName(spanName).
         setSource(sourceName).
-        setSpanId(Utils.convertToUuidString(zipkinSpan.id())).
-        setTraceId(Utils.convertToUuidString(zipkinSpan.traceId())).
+        setSpanId(spanId).
+        setTraceId(traceId).
         setStartMillis(zipkinSpan.timestampAsLong() / 1000).
         setDuration(zipkinSpan.durationAsLong() / 1000).
         setAnnotations(annotations).
         build();
 
+    if (zipkinSpan.tags().containsKey(SPAN_TAG_ERROR)) {
+      if (zipkinDataLogger.isLoggable(Level.FINER)) {
+        zipkinDataLogger.info("Span id :: " + spanId + " with trace id :: " + traceId +
+            " , failed with Error :: " + zipkinSpan.tags().get(SPAN_TAG_ERROR));
```

Comment:
Can we tweak this log message. Change `failed with Error` -> `includes error tag`.

This message makes it seem there is a failure within our proxy.